quizz/models.py

```diff
@@ -11,21 +11,24 @@ def __str__(self):
         repr1.maxstring = 200
         return "Quiz: {0} \n Description: {1}".format(self.name, repr1.repr(self.description))
 
+    class Meta:
+        verbose_name_plural = 'Quizzes'
+
 
 class Question(models.Model):
     text = models.CharField(max_length=400)
-    quiz = models.ForeignKey(Quiz, on_delete=models.CASCADE)
+    quiz = models.ForeignKey(Quiz, on_delete=models.CASCADE, related_name='questions')
 
     def __str__(self):
         repr1 = reprlib.Repr()
```

Comment:
(minor) Do you have name collision if you use 'repr' instead of 'repr1' ? I am somehow expecting for a 'repr2' if you name it that way.